### ISSUE TITLE
selinux module throws an error

### ISSUE THREAD

**Issue by jdkbx** [ultrabug/py3status#581]
If I enable the selinux module I get the following error message:

py3status: Instance `selinux`, user method `selinux_status` failed (AttributeError) selinux.py line 41. Please try to fix this and reload i3wm ...

the offending line in 
/usr/lib64/python2.7/site-packages/py3status/modules/selinux.py

```
39:    def selinux_status(self):
40:        try:
41:            if selinux.security_getenforce():
42:                selinuxstring = 'enforcing'
43:                color = self.py3.COLOR_GOOD
44:            else:
45:                selinuxstring = 'permissive'
46:                color = self.py3.COLOR_BAD
47:        except OSError:
48:            selinuxstring = 'disabled'
49:            color = self.py3.COLOR_BAD
```

Trying the same command in the python interpreter directly seems to work:

```
Python 2.7.12 (default, Oct  3 2016, 17:44:18) 
[GCC 5.4.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import selinux
>>> selinux.security_getenforce()
0
```

**Comment by tobes** [comment]
Thanks for reporting this. Is is due to python2 import logic, I've made a PR with the fix #582

**Comment by ultrabug** [comment]
merged, thanks for reporting @jdkbx !
